### PR TITLE
Removing fit_parameter bug in EVS14188* data

### PR DESCRIPTION
**Pull request by wathen**
#### Description of Work

Adds condition to check if fit_parameters is present in native fitbenchmark data format.

#### Testing Instructions

1. Check to see if example scripts run

### CODE REVIEW

File: fitbenchmarking/parsing/parse_fitbenchmark_data.py [hunk 1]
```diff
@@ -61,8 +61,9 @@ def __init__(self, fname):
         self._type = "FitBenchmark"
 
         self._starting_values = None
-        self._start_x = entries['fit_parameters']['StartX']
-        self._end_x = entries['fit_parameters']['EndX']
+        if 'fit_parameters' in entries:
```
Comment (Anders-Markvardsen) [inline fitbenchmarking/parsing/parse_fitbenchmark_data.py]: Thanks for spotting this and I agree it is reasonable to make StartX and EndX optional

**Comment by Anders-Markvardsen** [comment]
@wathen would you say this PR also fixes #163 ?

**Comment by wathen** [comment]
> @wathen would you say this PR also fixes #163 ?

Nope, that would need to be coded into the two classes seperately

**Comment by Anders-Markvardsen** [comment]
@wathen could you add a sentense to #163 of how you propose this issue fixed for future reference please

**Pull request merged**